contracts/src/interfaces/ITokenBridge.sol

```diff
@@ -48,7 +48,7 @@ struct SendTokensInput {
  * @param recipientGasLimit The amount of gas that will provided to the recipient contract on the destination chain,
  * which must be less than the requiredGasLimit of the message as a whole.
  * @param multiHopFallback In the case of a multi-hop transfer, the address where the tokens
- * are sent on the source chain if the transfer is unable to be routed to its final destination.
+ * are sent on the hub chain if the transfer is unable to be routed to its final destination.
```

Comment:
Not introduced in this PR, but the first line of this doccomment has a typo: `briding`